cnv/cnv_users_guide/cnv-viewing-ip-of-vm-vnic.adoc

```diff
@@ -8,7 +8,13 @@ The QEMU guest agent runs on the virtual machine and passes the IP address of at
 
 .Prerequisites
 
-* The QEMU guest agent must be xref:cnv-installing-qemu-guest-agent.adoc#cnv-installing-qemu-guest-agent[installed and running] on the virtual machine.
+. Verify that the guest agent is installed and running by entering the following command:
++
+----
+'systemctl status qemu-guest-agent'
```

Comment:
Commands need to start with $ or #, and unless it's required as part of the command syntax, it shouldn't be enclosed in single quotation marks.

Reply:
Fixed.